install-k8s.yml

```diff
@@ -3,6 +3,7 @@
     - masters
     - workers
   roles:
+#    - update-pkgs
```

Comment:
can we have this default?

Reply:
It sometimes takes a bit of time to download the packages due to the available network speed. Hope this is okay as we introduce the step as a default one in playbooks as it may impact the time taken for the jobs to finish.

Reply:
but this improves the stability and compliance of the cluster :)